app/jobs/generic_job.rb

```diff
@@ -56,6 +56,28 @@ def with_bulk_action_log
     end
   end
 
+  def with_items(druids, name:)
```

Comment:
Can you document how this method should be used? In particular, the success and failure lambdas.